.travis.yml

```diff
@@ -8,15 +8,24 @@ node_js:
   - "6"
 notifications:
   email:
-    - arnab@binary.com
+    #- arnab@binary.com
 branches:
   only:
     - master
     - development
+before_install:
+  # fire up xvfb on port :99.0
+  - "export DISPLAY=:99.0"
+  - "sh -e /etc/init.d/xvfb start"
+  # set the xvfb screen size to 1280x1024x16
+  - "/sbin/start-stop-daemon --start --quiet --pidfile /tmp/custom_xvfb_99.pid --make-pidfile --background --exec /usr/bin/Xvfb -- :99 -ac -screen 0 1280x1024x16"
+  # give Meteor some time to start
+  - "sleep 5"
 script:
-  - git config --global user.email "arnab@binary.com"
-  - git config --global user.name "Arnab Karmakar"
-  - yarn deploy-gh-pages
+  - yarn test
+  #- git config --global user.email "arnab@binary.com"
+  #- git config --global user.name "Arnab Karmakar"
+  #- yarn deploy-gh-pages
```

Comment:
@arnabk I am just testing one of the solutions I found, please don't merge this yet. I tested browser stack and found it to be working, I will revert my changes to that.